oggm/core/massbalance.py

```diff
@@ -354,7 +354,7 @@ def __init__(self, gdir, mu_star=None, bias=None, prcp_fac=None,
 
         super(ConstantMassBalance, self).__init__()
         self.mbmod = PastMassBalance(gdir, mu_star=mu_star, bias=bias,
-                                     prcp_fac=prcp_fac)
+                                     prcp_fac=prcp_fac, filename='climate_monthly', input_filesuffix='')
```

Comment:
I think the correct way here should be:

```filename=filename, input_filesuffix=input_filesuffix```

Otherwise the kwargs are not passed properly